src/main/features/dashboard/views/macro/caseHistory.njk

```diff
@@ -0,0 +1,76 @@
+{% from "./caseStatus.njk" import caseStatusForClaimant, caseStatusForDefendant %}
+{% from "timeRemaining.njk" import timeRemaining %}
+
+{% macro caseHistoryForClaimant(claim, url) %}
+  {% for state in claim.stateHistory %}
+    <div class="status-block">
+      {% if state.status === ClaimStatus.CLAIM_REJECTED %}
+        <h3 class="heading-small">{{ t('The defendant’s response') }}</h3>
+          <p>
+            {{ t('{{ defendantName }} has rejected your claim. We’ll send you a directions questionnaire – this is a form which you must complete to tell us more about the claim.',
+              { defendantName: claim.claimData.defendant.name }) }}
+          </p>
+          {# TODO: Check whether response PDF can be downloaded by claimant #}
+          {#<p>#}
+            {#<a href="{{ ResponsePaths.receiptReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Download their response') }}</a>#}
+          {#</p>#}
+      {% elif state.status === ClaimStatus.OFFER_ACCEPTED %}
+        <h3 class="heading-small">{{ t('Settle out of court') }}</h3>
+        <p>
+          {{ t('You’ve agreed to the offer made by {{ defendantName }} and signed an agreement to settle your claim.',
+            { defendantName: claim.claimData.defendant.name }) }}
+        </p>
+        <p>
+          {{ t('We’ve asked {{ defendantName }} to sign the agreement.',
+            { defendantName: claim.claimData.defendant.name }) }}
+        </p>
+      {% else %}
+        <p>
+          {{ caseStatusForClaimant(state.status, claim, url) }}
+        </p>
+      {% endif %}
+    </div>
+  {% endfor %}
+{% endmacro %}
+
+{% macro caseHistoryForDefendant(claim, url, isAfter4pm) %}
+  {% for state in claim.stateHistory %}
+    <div class="status-block">
+      {% if state.status === ClaimStatus.CLAIM_REJECTED %}
+        <h3 class="heading-small">{{ t('Your response to the claim') }}</h3>
+        <p>
+          {{ t('You have rejected the claim. The case will be reviewed by a judge and might go to court.') }}
```

Comment:
I'm pretty sure this should be "You've rejected the claim" as we're doing that else where but one to check with ronan

Reply:
Hmm, prototype has the following. I guess we are a bit inconsistent

<img width="822" alt="2d" src="https://user-images.githubusercontent.com/1724691/37462524-8b6099a2-2852-11e8-9c96-0bf7b3a7d908.png">

@RonanFitzgerald shall I change it to be `You've rejected the claim`?

Reply:
Either is fine but my personal preference is "you've" - there's no hard and fast rule about these "contractions" as they're known in our style guide.

Reply:
Will change that for consistency

@vianvi & @pragnesh-patel ☝️